src/main/webapp/app/exam/participate/exam-navigation-bar/exam-navigation-bar.component.html

```diff
@@ -27,6 +27,7 @@
                     <button class="btn w-100" [ngClass]="setExerciseButtonStatus(i)" (click)="changeExercise(i)">
                         <fa-icon [icon]="icon"></fa-icon>
                     </button>
+                    <p class="exerciseIndex">{{ i }}</p>
```

Comment:
```suggestion
                    <p class="exerciseIndex">{{ i + 1 }}</p>
```
I think it might make more sense if the exercise number displayed is starting with 1 instead with 0. Since maybe some students won't know what exercise 0 is supposed to be

Reply:
makes sense!